fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -1388,6 +1388,19 @@ private void initAggregateBuiltins() {
                 "",
                 "",
                 true, false, true, true));
+        addBuiltin(AggregateFunction.createBuiltin(FunctionSet.WINDOW_FUNNEL,
```

Comment:
Does we need add `DATEV2` override?

Reply:
`window_funnel` will be searched in `IS_NONSTRICT_SUPERTYPE_OF` mode so DATEV2 could be cast to DATETIMEV2 implicitly

Reply:
Ok, then do `AggregateFunctionWindowFunnel<DateV2Value<DateV2ValueType>, UInt32>` still need to register?

Reply:
Only `create_aggregate_function_window_funnel` is registered in function factory as a function creator. When we pass a window argument with date type, a function implement `AggregateFunctionWindowFunnel<DateV2Value<DateV2ValueType>, UInt32>` will be returned. So no extra things is registered